README.md

```diff
@@ -2,20 +2,35 @@
 
 # flink-faker
 
-flink-faker is an Apache Flink [table source](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/connectors/) 
-that generates fake data based on the [Java Faker](https://github.com/DiUS/java-faker) expression 
+flink-faker is an Apache Flink [table source](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/connectors/)
+that generates fake data based on the [Java Faker](https://github.com/DiUS/java-faker) expression
 provided for each column.
 
-Checkout this [demo web application](https://java-faker.herokuapp.com/) for some example Java Faker expressions.   
+Checkout this [demo web application](https://java-faker.herokuapp.com/) for some example Java Faker expressions.
 
-This project is inspired by [voluble](https://github.com/MichaelDrogalis/voluble). 
+This project is inspired by [voluble](https://github.com/MichaelDrogalis/voluble).
 
 ## Package
 
 ```shell script
 mvn clean package
 ```
 
+## Adding flink-faker to Flink SQL Client
+
+1. Download Flink from the [Apache Flink website](https://flink.apache.org/downloads.html).
+2. Download the flink-faker JAR from the [Releases](https://github.com/knaufk/flink-faker/releases) page (or [build it yourself](#package)).
+3. Put the downloaded jars under `FLINK_HOME/lib/`.
+4. (Re)Start a [Flink cluster](https://ci.apache.org/projects/flink/flink-docs-stable/docs/try-flink/local_installation/#step-2-start-a-cluster).
+5. (Re)Start the [Flink CLI](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sqlclient/).
+
+## Adding flink-faker to Ververica Platform
+
+1. [Setup Ververica Platform](https://docs.ververica.com/getting_started/installation.html).
+2. Download the flink-faker JAR from the [Releases](https://github.com/knaufk/flink-faker/releases) page (or [build it yourself](#package)).
```

Comment:
Actually that step is not needed. You can also reference the path to the release on Github directly in 3. It supports http(s).